src/com/xilinx/rapidwright/interchange/DeviceResourcesWriter.java

```diff
@@ -333,22 +337,24 @@ public static void writeDeviceResourcesFile(String part, Device device, CodePerf
         	}
         }
 
+        Map<String, Pair<String, EnumSet<IOStandard>>> macroCollapseExceptionMap = 
+                EDIFNetlist.macroCollapseExceptionMap.getOrDefault(series, Collections.emptyMap());
```

Comment:
```suggestion
        Map<String, Pair<String, EnumSet<IOStandard>>> macroCollapseExceptionMap = 
        EDIFNetlist.macroCollapseExceptionMap.getOrDefault(series, Collections.emptyMap());
```

Reply:
This indent is preferred to indicate a line continuation from the previous line.

Reply:
Oops, sorry!